pages/models/generations/Sources/VSC/PhasorGridFollowingVSC/PhasorPQ1/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page 
+title: Phasor-PQ1 Grid Following Voltage Source Converter 
+tags: [Opensource, Phasor, voltage source, converter, wind, pv, hdvc, dynawo, STEPSS] 
+date: 26/06/2024 
+last-updated: 26/06/2024
+id: #165
+authors: Carlos Alegre (eRoots)
+reviewers: Eduardo Prieto Araujo (UPC), Josep Fanals Batllori (eRoots)
+---
+
+
+## Context
+
+The Phasor-PQ1 Grid Following Voltage Source Converter (VSC) model [[1]](#1) is derived from the Full-Phasor described [here](../). 
+
+## Model use, assumptions, validity domain and limitations
+
+The model can be used to fasten the simulations for low-frequency phenomena studies. 
+
+The assumptions made starting from the Full-Phasor model are the following:
+
+* The current loop is removed, and the VSC current is set to be directly the output of the outer loop.
+
+* The outer loop is substituted by a first-order transfer function with a given time constant.
+
+The model shows the best performance out of all the models when using big time-steps (the breakpoint is around $$\Delta t = 10^3$$), making it the best alternative for studies with the lowest frequency dynamics or for cases that require the fastest performance. 
+
+The error is considerably higher than any of the more complete models for smaller time steps, which means any other model is preferable.
+
+## Model description
+
+The diagram of the complete model is shown in the following figure.
+
+<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; padding:4px 0;">
+<img src="{{ '/pages/models/generations/Sources/VSC/PhasorGridFollowingVSC/PhasorPQ1/PhasorPQ1.svg' | relative_url }}"
+     alt="Phasor Models Comparison"
+     style="float: center; margin-right: 10px; width: 600px;" />
+</div>
+<div align = 'center'>
+Figure 1: Phasor-PQ1 Model Block Diagram <a href="#1">[1]</a>
+</div>
+<br>
+
+The following subsections will describe the changes with respect to the Full-Phasor model.
+
+### Current loop
+
+The current loop is completely removed, which means $$i^{qd*} = i^{qd*}_c $$.
+
+### Outer loop
+
```

Comment:
same inputs/output variables as on the big complete diagram? if not, add an input/output diagram.